config.go

```diff
@@ -45,48 +45,50 @@ type ConfigSection interface {
+	GithubPRCreatorOrg      string                    `yaml:"github_pr_creator_org" bson:"github_pr_creator_org" json:"github_pr_creator_org"`
+	GithubOrgs              []string                  `yaml:"github_orgs" bson:"github_orgs" json:"github_orgs"`
+	HostInit                HostInitConfig            `yaml:"hostinit" bson:"hostinit" json:"hostinit" id:"hostinit"`
+	HostJasper              HostJasperConfig          `yaml:"host_jasper" bson:"host_jasper" json:"host_jasper" id:"host_jasper"`
+	Jira                    JiraConfig                `yaml:"jira" bson:"jira" json:"jira" id:"jira"`
+	JIRANotifications       JIRANotificationsConfig   `yaml:"jira_notifications" json:"jira_notifications" bson:"jira_notifications" id:"jira_notifications"`
+	Keys                    map[string]string         `yaml:"keys" bson:"keys" json:"keys"`
+	KeysNew                 util.KeyValuePairSlice    `yaml:"keys_new" bson:"keys_new" json:"keys_new"`
+	LoggerConfig            LoggerConfig              `yaml:"logger_config" bson:"logger_config" json:"logger_config" id:"logger_config"`
+	LogPath                 string                    `yaml:"log_path" bson:"log_path" json:"log_path"`
+	Notify                  NotifyConfig              `yaml:"notify" bson:"notify" json:"notify" id:"notify"`
+	Plugins                 PluginConfig              `yaml:"plugins" bson:"plugins" json:"plugins"`
+	PluginsNew              util.KeyValuePairSlice    `yaml:"plugins_new" bson:"plugins_new" json:"plugins_new"`
+	PprofPort               string                    `yaml:"pprof_port" bson:"pprof_port" json:"pprof_port"`
+	Providers               CloudProviders            `yaml:"providers" bson:"providers" json:"providers" id:"providers"`
+	RepoTracker             RepoTrackerConfig         `yaml:"repotracker" bson:"repotracker" json:"repotracker" id:"repotracker"`
+	Scheduler               SchedulerConfig           `yaml:"scheduler" bson:"scheduler" json:"scheduler" id:"scheduler"`
+	ServiceFlags            ServiceFlags              `bson:"service_flags" json:"service_flags" id:"service_flags" yaml:"service_flags"`
+	Slack                   SlackConfig               `yaml:"slack" bson:"slack" json:"slack" id:"slack"`
+	SpawnHostsPerUser       *int                      `yaml:"spawn_hosts_per_user" bson:"spawn_host_per_user" json:"spawn_hosts_per_user"`
+	Splunk                  send.SplunkConnectionInfo `yaml:"splunk" bson:"splunk" json:"splunk"`
+	SuperUsers              []string                  `yaml:"superusers" bson:"superusers" json:"superusers"`
+	Triggers                TriggerConfig             `yaml:"triggers" bson:"triggers" json:"triggers" id:"triggers"`
+	Ui                      UIConfig                  `yaml:"ui" bson:"ui" json:"ui" id:"ui"`
+	UnexpirableHostsPerUser *int                      `yaml:"unexpirable_hosts_per_user" bson:"unexpirable_hosts_per_user" json:"unexpirable_hosts_per_user"`
```

Comment:
Can we make these new values just intergers?

Reply:
I think we need to differentiate between null and 0 for these

Reply:
nah, not really? what do we do when it's nil? default to a constant?

Reply:
Yep it defaults to 1 if nil

Reply:
I think we should just let it default to 0, and take out the pointer.

Reply:
I'm not a fan of that. 0 is a valid value to have but should not be the default

Reply:
I made these a pointer because https://github.com/evergreen-ci/evergreen/blob/master/rest/model/admin.go#L174 was panicking. Alternatively I can check if `val.Type() == int` or something but I wasn't sure if there was a reason we checked this.

Reply:
Lets remove the pointer. I don't think we should set a default if something is 0

Reply:
But then how do you differentiate between 0 and the default (1)?